README.md

```diff
@@ -1,5 +1,7 @@
 # Opdracht beschrijving
```

Comment:
Compliment met de kleine letters in je branch naam. Compliment met het aanmaken van commits na elke grote verandering.